### ISSUE TITLE
Ability to add rounds/data from the app

### ISSUE THREAD

**Issue by ericgio** [ericgio/weekday-golf#19]
Right now the data entry process involves:

1. Makinde entering scores in the UDisc app
2. Makinde posting a screenshot of the round to the thread
3. Someone manually entering the data from the screenshot to the google spreadsheet

Would be nice to cut out a bunch of those steps and just be able to enter data directly in the app.

**Comment by makinde** [comment]
yeah buddy. talked to ola a bit about this. Would be nice to have some simple backend to write data to for this. The spreadsheet /could/ could, but feels really contrived. Also, this would involve building a good UI. "big" project, relatively speaking. Would love to see it someday.

**Comment by makinde** [comment]
working on this

**Comment by makinde** [comment]
BTW, if we add any sort of authentication, we'll prob need HTTPS to work with google/FB/auth0. Another reason to consider #16 . Already running at dotherest.com :)
